Core/PixelEvent.swift

```diff
@@ -661,48 +661,20 @@ extension Pixel {
         case pixelExperimentEnrollment
```

Comment:
⚠️ Listed as new in the task, but not removed.

Reply:
@dus7, pixelExperimentEntrollment can stay there since it isn't fired anymore and it is more related to the experiment logic itself than new Settings. Does it make sense to you?